pkg/exasol-rest-api/application.go

```diff
@@ -0,0 +1,19 @@
+package exasol_rest_api
```

Comment:
> Every package should have a package comment, a block comment preceding the package clause. For multi-file packages, the package comment only needs to be present in one file, and any one will do. 

https://golang.org/doc/effective_go#commentary

Reply:
This is not a reusable module and is not intended to be used as a library. So the comment will be just for a checkmark, like for getters